sky/cli.py

```diff
@@ -2918,6 +2919,7 @@ def _down_or_stop(name: str):
             except (exceptions.NotSupportedError,
                     exceptions.ClusterNotUpError) as e:
                 message = str(e)
+                sys.exit(1)
```

Comment:
We don't want a failure for a single cluster to abort the termination for the other clusters. Since we are running the terminations in parallel, how about we gather all the failures and show them in a table at the end and return non-zero for any failure happens during the termination **at the end**?